en/snap/platform/overview.md

```diff
@@ -65,57 +67,59 @@ On **Sandbox** mode:
 | Error Redirect URL | [your-site-url] |
 | Unfinish Redirect URL | [your-site-url] |
 
-3. Choose __Settings - Access Keys__.
+3. Choose __Settings --> Access Keys__.
 
-	Copy Midtrans __Merchant ID__ dan __Server Key__ (Will be used for the next step).
+	Copy Midtrans __Merchant ID__ and __Server Key__ (Will be used for the next step).
 
 	![access key](./../../../asset/image/sirclo-2.png)
 
-4. Login to your [Shopify Store](https://www.shopify.com/login) 
+4. Login to your [Shopify Store](https://www.shopify.com/login).
 
-5. In your Shopify Store admin page go to menu __Settings - Payment providers__ 
+5. On your Shopify Store- admin page, go to __Settings --> Payment providers__.
 	
 	![Settings menu](./../../../asset/image/shopify-1.png ':size=400')
 	![Payment providers menu](./../../../asset/image/shopify-2.png ':size=400')
 
-6. Under __Third-party providers__ menu, click __Choose third-party provider__.
+6. Under __Third-party providers__, click __Choose third-party provider__.
 	![Third-party providers](./../../../asset/image/shopify-3.png ':size=400')
 
-7. Search __Midtrans__ and click it.
+7. Enter **Midtrans** in search bar and click **Search**.
 	![Search Midtrans](./../../../asset/image/shopify-4.png ':size=400')
 
-8. Complete the configuration by filling in your __Merchant ID__ & __server key__. Then click __Activate__. 
+8. Complete the configuration by filling in your __Merchant ID__ and __server key__. Then click __Activate__. 
 
-	__Note:__
+ ?>***Note:***
 
-	- Leave __Enable test mode__ checkbox unchecked to allow real transaction.
-	- Check the __Enable test mode__ checkbox if you want to test transaction without actual payment (using test credentials provided by Midtrans).
+  - Uncheck __Enable test mode__ checkbox to allow real transaction.
+  - Check the __Enable test mode__ checkbox if you want to test transaction without actual payment (using test credentials provided by Midtrans).
 
-	Please note that __server key__ for sandbox/testing is different from production. You may obtain your Merchant ID & server key from Midtrans MAP (after logging in) from the following links:
-	
-	- For sandbox/testing: [here](https://dashboard.sandbox.midtrans.com/settings/config_info)
-	- For production/real transaction: [here](https://dashboard.midtrans.com/settings/config_info)
+  ?>***Note:*** __Server key__ for *Sandbox*/testing is different from production. You may obtain your *Merchant ID* and *server key* from Midtrans MAP (after logging in) from the following links:
 
-	![shopify](./../../../asset/image/shopify-5.png ':size=400')
+  - For *Sandbox*/testing: [here](https://dashboard.sandbox.midtrans.com/settings/config_info)
+  - For *Production*/real transaction: [here](https://dashboard.midtrans.com/settings/config_info)
```

Comment:
This looks off to me.  If you cannot improve how it is rendered, please leave the rendering format unchanged

<img width="791" alt="Screen Shot 2020-11-20 at 12 56 05" src="https://user-images.githubusercontent.com/13027142/99764496-c98f4880-2b2f-11eb-8077-a8abd9c563c6.png">